src/core/qgsproviderregistry.h

```diff
@@ -107,6 +107,14 @@ class CORE_EXPORT QgsProviderRegistry
      */
     int providerCapabilities( const QString &providerKey ) const;
 
+    /**
+     * Returns the components (e.g. file path, layer name) of a provider uri
+        \param providerKey identificator of the provider
```

Comment:
Fun fact: identificator  isn't a word :stuck_out_tongue: (but you didn't start this trend, so I'll let it pass!)